### ISSUE TITLE
--ignore-paths invalid option in .puppet-lint.rc

### ISSUE THREAD

**Issue by SanderKnape** [rodjek/puppet-lint#832]
Hello,

When I use the --ignore-paths flag in the .puppet-lint.rc file, I get the following error: `puppet-lint: invalid option: --ignore-paths "stdlib/*"`. The line looks like this: `--ignore-paths "stdlib/*"`

I also tried to do `--ignore-paths="stdlib/*"`, which doesn't give an error, but then the directory is not ignored.

What is the proper way to do this in the .puppet-lint.rc file?

**Comment by josa42** [comment]
@SanderKnape 

I works for me without the quotation marks: `--ignore-paths=stdlib/*`.

**Comment by RyanJarv** [comment]
`--ignore-paths=stdlib/*` seems to work here but only when it's the first option in the file, fails when it's in any other order.

**Comment by usev6** [comment]
What @josa42 and @RyanJarv suggested should indeed work in a config file. The versions with quotes like ```--ignore-paths "stdlib/*"``` are only useful for passing arguments on the command line. I opened https://github.com/rodjek/puppet-lint/pull/878 in the hope it enhances the relevant documentation a bit.

> --ignore-paths=stdlib/* seems to work here  but only when it's the first option in the file, fails when it's in any other order.

I wasn't able to reproduce a problem when there are other options before ```--ignore-paths=stdlib/*```. @RyanJarv : could you maybe describe a setup that shows the problem?

This works for me. The following code assumes there is no .puppet-lint.rc file at all:
```
$ mkdir -p test_gh-832/gh{1,2}
$ cd test_gh-832
$ echo '$foo-bar = 42' > gh1/test.pp   ## generating a file that has a problem
$ echo '$foo-bar = 42' > gh2/test.pp   ## generating another file that has a problem
$ puppet-lint .
./gh2/test.pp - WARNING: variable contains a dash on line 1
./gh1/test.pp - WARNING: variable contains a dash on line 1
$ echo '--no-hard_tabs-check' >> .puppet-lint.rc   ## should be the same with ~/.puppet-lint.rc
$ echo '--ignore-paths=./gh1/*' >> .puppet-lint.rc
$ puppet-lint .
./gh2/test.pp - WARNING: variable contains a dash on line 1
```

**Comment by RyanJarv** [comment]
To be honest I was a bit confused about how my repo was set up when I was testing this so may have just been confused there. I would go ahead and ignore my comment since I tossed the repo for other reasons and may not get the chance to retry it. I'll go ahead and update this though if I can reproduce it again.

**Comment by usev6** [comment]
Great. To me it looks like this issue can be closed. There is always the 'reopen' option if a setup that triggers the described problem is reported.

**Comment by rodjek** [comment]
Closing this for now. As @usev6 mentioned, please don't hesitate to reopen the issue if the issue persists
